bundles/bundles.go

```diff
@@ -204,10 +213,16 @@ func (e OCIImageExtractor) Install(config *BundleConfig) error {
 	if err != nil {
 		return err
 	}
+
+	platform := config.Platform
+	if len(platform) == 0 {
+		platform = utils.GetCurrentPlatform()
+	}
```

Comment:
Sounds to me like we never want to use `config.Platform` directly. Could you instead add a function, e.g. `BundleConfig.Platform()` that will do this operation?

Reply:
can we also have tests for the added code?

Reply:
sure i will make changes

Reply:
@mauromorales can you review again?